### ISSUE TITLE
VertexAI models to grade metrics returns error 400

### ISSUE THREAD

**Issue by romaintoub** [promptfoo/promptfoo#447]
Hi,

I've tested the default prompt config file and it works well with Vertex AI models as providers but when I use them as evaluators for testing like here: https://promptfoo.dev/docs/configuration/expected-outputs/model-graded#overriding-the-llm-grader
then I am getting the following error:

```
| [FAIL] Error 400: 1 instance(s) is allowed per prediction. Actual: 2 |
```
I've tried to override the LLM grader in different ways and got the same error. 

`promptfooconfig.yaml`:
```
# Learn more: https://promptfoo.dev/docs/configuration/guide
description: 'my description'

prompts:
  - "Write a tweet about {{topic}}"
  - "Write a very concise, funny tweet about {{topic}}"

providers:
  - vertex:gemini-pro
  - vertex:chat-bison

defaultTest:
  options:
    provider: vertex:gemini-pro

tests:
  - vars:
      topic: bananas

  - vars:
      topic: avocado toast
    assert:
      # For more information on assertions, see https://promptfoo.dev/docs/configuration/expected-outputs
      - type: icontains
        value: avocado
      - type: javascript
        value: 1 / (output.length + 1)  # prefer shorter outputs

  - vars:
      topic: new york city
    assert:
      # For more information on model-graded evals, see https://promptfoo.dev/docs/configuration/expected-outputs/model-graded
      - type: llm-rubric
        value: ensure that the output is funny
```

When I run `promptfoo eval`:
![image](https://github.com/promptfoo/promptfoo/assets/64668750/369cb65a-2c08-41f0-9f3a-c01cae53390d)

thanks for your help

**Comment by typpo** [comment]
The `llm-rubric` prompt (and other model-graded eval prompts) are written in OpenAI chat format.  This is usually ok, even for alternative models, because most APIs mimic OpenAI's format.  But Vertex is an exception!  PR #450 should solve this issue for Vertex.

**Comment by romaintoub** [comment]
thank you for your help! very much appreciated
I'd love to help on this great project but I need to improve on node.js first ahah

**Comment by typpo** [comment]
This fix is published in 0.40.0
